radicle-node/src/service.rs

```diff
@@ -612,7 +609,8 @@ where
                 } else {
                     return Ok(false);
                 }
-                self.process_inventory(&message.inventory, *node, git);
+                self.process_inventory(&message.inventory, *node, git)
+                    .unwrap();
```

Comment:
Is this known to never panic?  Maybe a comment why if so would help

Reply:
Nah, we're not there yet, ie. there are *lots* of unwraps in the codebase that need to be handled correctly. We'll have to do a pass at some point and fix them.